ccm_web/client/src/api.ts

```diff
@@ -0,0 +1,21 @@
+import getLTIKey from './utils/getLTIKey'
```

Comment:
is this file a collection of all api calls for mergeSection, create sections etc on client side? This is a good organization

Reply:
Yep, that's the idea!

Reply:
few comments on file usefulness/purpose will be helpful?